test/link-check/linkCompare.test.js

```diff
@@ -0,0 +1,482 @@
+import fs from 'fs';
+import { expect } from '@esm-bundle/chai';
+import { stub } from 'sinon';
+import { compare, compareLink, compareLinks, extractLinks } from '../../link-check/linkCompare.js';
+import createMocks from './createMocks.js';
+
+const { pathname } = new URL('.', import.meta.url);
+
+const linkURLs = [
+  'https://www.adobe.com/creativecloud.html',
+  'https://www.adobe.com/products/photoshop.html',
+  'https://www.adobe.com/products/illustrator.html',
+  'https://www.adobe.com/products/premiere.html',
+  'https://www.adobe.com/products/acrobat.html',
+];
+
+describe('Link comparison', () => {
+  before(async () => {
+    let links1 = '# Links';
+    linkURLs.forEach((link) => {
+      links1 += `\n- [Adobe](${link})`;
+    });
+    createMocks(links1, 'links1');
+    let links2 = '# Links';
+    linkURLs.slice().reverse().forEach((link) => {
+      links2 += `\n- [Adobe](${link})`;
+    });
+    createMocks(links2, 'links2');
```

Comment:
The outcome of this will be "create fils with the linkURLs if they don't exist, return the files that are there if they do exist" correct? A little conflicted since we are adding files to the repo that this really only needed to be done once?

Reply:
I moved this out of the test file, could be useful in the future if we ever need to modify the tests.